docs/contributor/benchmarks/cls-performance.md

```diff
@@ -0,0 +1,139 @@
+shoot--kyma-stage--d7ca774-cpu-worker-0-z3-5dc4c-mz8zd   Ready    <none>   23h   v1.31.7   10.250.0.7    <none>        Garden Linux 1592.8   6.6.84-cloud-amd64   containerd://1.7.23
+shoot--kyma-stage--d7ca774-cpu-worker-0-z3-5dc4c-rnq2f   Ready    <none>   22h   v1.31.7   10.250.0.12   <none>        Garden Linux 1592.8   6.6.84-cloud-amd64   containerd://1.7.23
+```
+
+### cluster details
+```sh
+Client Version: v1.33.0
+Kustomize Version: v5.6.0
+Server Version: v1.31.7
+WARNING: version difference between client (1.33) and server (1.31) exceeds the supported minor version skew of +/-1
+```
+
+### Log Pipeline
+
+The test uses a basic LogPipeline with an enabled OTLP input and a default output to CLS:
+
+[log-pipeline configuration](assets/otel-input.yaml)
+
+
+# Goal
+
+Validate if a default Telemetry module deployment can saturate one CLS-Instance (large).
+Large instances support up to 30.000 logs per second if the logs are ingested using [OTLP](https://pages.github.tools.sap/perfx/cloud-logging-service/plans-and-prices/#storage-limitations).
+
+# Direct Setup
+
+The following setup establishes a baseline for the performance of telemetry gen. CLS is used as a sink for the logs generated by telemetry gen.
+
+Log generation:
+- Payload: ~2k (plus metadata)
+- [telemetry-gen configuration](assets/telemetry-gen-direct-to-cls.yml)
+
+```mermaid
+flowchart LR
+    A[telemetrygen] --> B[CLS-service]
+
+    style A fill:#f9f,stroke:#333,stroke-width:1px
+    style B fill:#bbf,stroke:#333,stroke-width:1px
+```
+
+(unit: logs per second)
+
+| **workers ↓ / replicas →** | 1      | 5       | 10      |
+| -------------------------- | ------ | ------- | ------- |
+| **1**                      | ~475   | ~2.260  | ~4.600  |
+| **5**                      | ~2.300 | ~11.600 | ~22.600 |
+| **10**                     | ~4.500 | ~22.500 | ~43.600 |
+
+## Result
+To achieve more than 30000 logs per second, 10 replicas with 10 workers each are required.
+
+
+# Telemetry Module Setup
+
+The following setup measures the performance of the Telemetry module. CLS is used as a sink in the configured LogPipeline. The Telemetry module is configured to use the OTLP input. During the test, we scale the number of replicas of the telemetry-log-gateway to measure the performance of the Telemetry module.
+
+Log generation:
+- Payload: ~2k (plus metadata)
+- Workers: 10
+- Replicas: 10
+- [telemetry-gen configuration](assets/telemetry-gen-using-log-pipeline.yaml)
+
+```mermaid
+flowchart LR
+    A[telemetrygen] --> C[telemetry-logs-gateway]
+    C[telemetry-logs-gateway] --> B[CLS-service]
+
+    style A fill:#f9f,stroke:#333,stroke-width:1px
+    style B fill:#bbf,stroke:#333,stroke-width:1px
+    style C fill:#bfb,stroke:#333,stroke-width:1px
+
+```
+
+| Gateway Replicas | Throughput Msg / s | Enqueue failed Msg / s | Memory per instance (MB) | CPU per instance | Bytes transmitted (MB/s) | Bytes received (MB/s) | istio Memory | istio cpu |
```

Comment:
Can you please add the metric you used to measure the throughput so that there is no confusion when looking at this sometime in future.